recipes/sunpy/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set version = "0.6.4" %}
+
+package:
+  name: sunpy
+  version: {{version}}
+
+source:
+  fn: sunpy-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/source/s/sunpy/sunpy-{{version}}.tar.gz 
+  md5: 45da6a4328fc5c85c717686f8aae2d23
+
+build:
+  number: 0
+  script: python setup.py install --offline --no-git
+  skip: True  # [py>27]
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >1.7.1
+    - astropy >=1.0
+    - scipy
+    - matplotlib
+    - pandas
+    - mock
+
+  run:
+    - python
+    - numpy >1.7.1
+    - astropy >=1.0
+    - scipy
+    - pandas >=0.12.1
+    - matplotlib
+    - suds-jurko
+    - glymur
+    - sqlalchemy
+    - beautiful-soup
+    - requests
+    - bsddb  # [unix]
+    - pytest
+    - scikit-image
+    - wcsaxes
+
+test:
+  # Python imports
+  imports:
+    - sunpy
+    - sunpy.cm
+    - sunpy.coordinates
+    - sunpy.data
+    - sunpy.data.test
+    - sunpy.data.test.waveunit
+    - sunpy.database
+    - sunpy.database.tests
+    - sunpy.image
+    - sunpy.image.tests
+    - sunpy.instr
+    - sunpy.instr.iris
+    - sunpy.io
+    - sunpy.io.tests
+    - sunpy.lightcurve
+    - sunpy.lightcurve.sources
+    - sunpy.lightcurve.tests
+    - sunpy.map
+    - sunpy.map.sources
+    - sunpy.map.tests
+    - sunpy.net
+    - sunpy.net.hek
+    - sunpy.net.hek2vso
+    - sunpy.net.helio
+    - sunpy.net.jsoc
+    - sunpy.net.tests
+    - sunpy.net.vso
+    - sunpy.physics
+    - sunpy.physics.transforms
+    - sunpy.roi
+    - sunpy.spectra
+    - sunpy.spectra.sources
+    - sunpy.spectra.tests
+    - sunpy.sun
+    - sunpy.sun.tests
+    - sunpy.tests
+    - sunpy.tests.tests
+    - sunpy.time
+    - sunpy.time.tests
+    - sunpy.util
+    - sunpy.util.tests
+    - sunpy.visualization
+    - sunpy.wcs
+    - sunpy.wcs.tests
+
+about:
+  home: http://www.sunpy.org/
+  license:  BSD License
```

Comment:
Do you know which version? `BSD 2-Clause` or `BSD 3-Clause`? (You can drop the extra `License` BTW. That is a redundant boiler plate from `conda skeleton`.)